### ISSUE TITLE
standard_init_linux.go:211: exec user process caused "exec format error"

### ISSUE THREAD

**Issue by jfosback** [openzipkin/zipkin#3084]
## Describe the Bug

When pulling the image openzipkin/zipkin:2.21.1 to our Kubernetes machines, I'm getting the following error:

standard_init_linux.go:211: exec user process caused "exec format error"

Here is the uname -a:
 
Linux zipkin-5f956c997d-njn8c 4.19.56-1.el7.x86_64 #1 SMP Wed Jun 26 23:17:23 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

Here is the output of describing the pod (mirror names, host names, namespaces changed to make this generic):

```
Events:
  Type     Reason     Age               From                           Message
  ----     ------     ----              ----                           -------
  Normal   Scheduled  <unknown>         default-scheduler              Successfully assigned my-namespace/zipkin-7f86b97487-xcblh to my-machine.com
  Normal   Generated  16s               appcertfetcher                 Application certificate generated
  Normal   Pulling    16s               kubelet, my-machine.com        Pulling image "my-mirror.com/openzipkin/zipkin:2.21.1"
  Normal   Pulled     10s               kubelet, my-machine.com        Successfully pulled image "my-mirror.com/openzipkin/zipkin:2.21.1"
  Normal   Created    9s (x2 over 10s)  kubelet, my-machine.com        Created container zipkin
  Normal   Pulled     9s                kubelet, my-machine.com        Container image "my-mirror.com/openzipkin/zipkin:2.21.1" already present on machine
  Normal   Started    8s (x2 over 10s)  kubelet, my-machine.com        Started container zipkin
  Warning  BackOff    7s (x2 over 8s)   kubelet, my-machine.com        Back-off restarting failed container
```

## Steps to Reproduce

Attempt to pull the image openzipkin/zipkin:2.21.1.  Image pulls successfully, but then results in the above error message when it attempts to start.

## Expected Behaviour

I would expect the image to start up successfully in this environment.  I'm unclear why these machines are giving this error consistently.  The error does not appear to be tied to this specific version of Zipkin.

**Comment by jorgheymans** [comment]
our resident docker expert @llinder may have an idea about this ?

**Comment by jfosback** [comment]
This appears to be resolved in 2.21.5.
